tests/test_modeling_tf_common.py

```diff
@@ -312,7 +312,7 @@ def test_pt_tf_model_equivalence(self):
                 print(config)
                 print(inputs_dict)
                 print(pt_inputs_dict)
-            self.assertLessEqual(max_diff, 2e-2)
+            self.assertLessEqual(max_diff, 4e-2)
```

Comment:
think we should update the `if max_diff >= 4e-2:` as well.

Reply:
Didn't see it, thanks for catching that mistake!